chore: create SECURITY.md (#6244)
prisma/prisma: SECURITY.md

# Security Policy

> Please send an email to [security@prisma.io](mailto:security@prisma.io) describing what you have found.
> Please do not raise an issue in this repository or publicize your concern in any other place without giving us adequate time to investigate first.

Author: Let's use the same language we use in README then:
```suggestion
If you have a security issue to report, please contact us at [security@prisma.io](mailto:security@prisma.io).
```
